aasemble/django/apps/api/v2/views.py

```diff
@@ -68,6 +68,7 @@ def perform_create(self, serializer):
 class SnapshotViewSet(mixins.CreateModelMixin,
                       mixins.RetrieveModelMixin,
                       mixins.DestroyModelMixin,
+                      mixins.UpdateModelMixin,
```

Comment:
You can replace all of these with `class SnapshotViewSet(viewsets.ModelViewSet)`. I listed these individually to avoid UpdateModelMixin, since I didn't want to allow changing anything about snapshots.